src/loaders/tiffPyramidLoader.js

```diff
@@ -13,11 +15,26 @@ export default class TiffPyramidLoader {
     const imageWidth = firstFullImage.ImageWidth;
     const imageHeight = firstFullImage.ImageLength;
     const tileSize = firstFullImage.TileWidth;
+    const bitsPerSample = firstFullImage.BitsPerSample;
+    let dtype;
+    if (bitsPerSample instanceof Uint8Array) {
+      dtype = '<u1';
+    }
+    if (bitsPerSample instanceof Uint16Array) {
+      dtype = '<u2';
+    }
+    if (bitsPerSample instanceof Uint32Array) {
+      dtype = '<u4';
+    }
+    if (bitsPerSample instanceof Float32Array) {
+      dtype = '<f4';
+    }
```

Comment:
I think it would be my preference to pass around `"Uint8"`, `"Uint16"`, `"Uint32"`, and `"Float32"` since we are in javascript-land, but I don't feel too strongly about this. Maybe @mccalluc can chime in..

Reply:
Ahh forget it. These are from the zarr store..

Reply:
Yea I just wanted to unify our convention here.